extensions/vscode/src/utils/url.test.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2025 by Posit Software, PBC.
+
+import { describe, expect, test } from "vitest";
+import { formatURL, normalizeConnectURL } from "src/utils/url";
+
+describe("URL util", () => {
```

Comment:
Love seeing all of these unit tests ❤️ Would it make sense to also add at least one `https://deeply.nested.sub.domain.connect.company.com/connect`?